setup

```diff
@@ -269,6 +270,9 @@ case "$1" in
 	replaceIDs /etc/subuid containers   ${CONTAINER_IDS}
 	replaceIDs /etc/subgid containers   ${CONTAINER_IDS}
 
+    # Create a QM seccomp.json before start the QM daemon
+    $("${QM_SECCOMP_TOOL}")
```

Comment:
@dougsland 
I see this 
https://github.com/containers/qm/pull/362/files#diff-055b3acf274095b1eb61f3d7e44115449460dbf49660abaed65484e771574faeR34

It will be called in here, based on podman unit, file

Reply:
I thought I already fixed that one, I will take a look again, thanks @Yarboa

Reply:
@Yarboa also added Requires containers-common thanks!!